osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerPlaylist.cs

```diff
@@ -213,6 +233,17 @@ private void assertItemInHistoryListStep(int playlistItemId, int visualIndex)
             });
         }
 
+        private void assertQueueTabCount(int count)
+        {
+            string queueTabText = count > 0 ? $"Queue ({count})" : "Queue";
+            AddUntilStep($"Queue tab shows \"{queueTabText}\"", () =>
+            {
+                return this.ChildrenOfType<OsuTabControl<MultiplayerPlaylistDisplayMode>.OsuTabItem>()
+                           .First()
+                           .Text.Text.Equals(queueTabText);
```

Comment:
You can check the text of the queue directly using another `ChildrenOfType` query:
```suggestion
                return this.ChildrenOfType<OsuTabControl<MultiplayerPlaylistDisplayMode>.OsuTabItem>()
                           .Single(t => t.Value == MultiplayerPlaylistDisplayMode.Queue)
                           .ChildrenOfType<OsuSpriteText>().Single().Text == queueTabText;
```
Never expose a property as `protected`/`public` for testing purposes if it's part of the hierarchy of that component, as you can use `ChildrenOfType` for that instead.

Reply:
Whoops, this definitely makes more sense.